obsreport/obsreport.go

```diff
@@ -34,6 +34,8 @@ var (
 	gLevel = configtelemetry.LevelBasic
 
 	okStatus = trace.Status{Code: trace.StatusCodeOK}
+
+	latencyDistribution = view.Distribution(0.001, 0.0025, 0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1)
```

Comment:
Should these be configurable by user? Or have a default and a user can define them? 

Can I also ask what is the values mean? Like is it tracking milliseconds, microseconds? Is worth adding some multiple of time here if it the case?

Reply:
Ideally we will use a value that works for most use-cases.  At least to start, I wouldn't expect it to be configurable.
The values are in the unit of the metric, which is UnitSeconds.